recipes/pytest-executable/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pytest_executable" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: pytest-executable
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 316b7982bcdb923ad6b8453f54b348bf81328dcdcecad212034acdf592bd3a4d
+
+build:
+  number: 0
+  include_recipe: False
```

Comment:
Unless there is a compelling reason, please do not use `include_recipe: False`

Reply:
There was a reason when I created the recipe locally, it's no longer the case so I can remove it.